pkgs/by-name/la/labelle/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  qt6,
+  makeDesktopItem,
+  copyDesktopItems,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "labelle";
+  version = "1.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "labelle-org";
+    repo = pname;
```

Comment:
```suggestion
    repo = "labelle";
```

Reply:
Where is this in the manual?

I am curious, since I am removing references to pname in Nixpkgs.

Reply:
There are several places, for example https://nixos.org/manual/nixpkgs/stable/#python-package-using-setuptools-rust

Could you please tell me why it's beneficial to change repo to a fixed string? I still need the pname on the top level, otherwise it doesn't build...

Reply:
> There are several places, for example https://nixos.org/manual/nixpkgs/stable/#python-package-using-setuptools-rust
> 
> Could you please tell me why it's beneficial to change repo to a fixed string? I still need the pname on the top level, otherwise it doesn't build...

Semantically `repo` and `pname` are not equal. For example I can override derivation and change `pname`, but `repo` will be broken in such case.

Reply:
Thank you for the explanation. Changed it with the latest push.